collectors/ebpf.plugin/ebpf.c

```diff
@@ -236,6 +242,9 @@ static void ebpf_exit(int sig)
         freez(shm_pid);
     }
 
+    ebpf_close_cgroup_shm();
+
+    ebpf_clean_cgroup_pids();
```

Comment:
Should we only do this if cgroup is enabled?

Reply:
The functions have tests to  clean variables only when they were set, they are like `freez`, so to call them won't be a problem.